spec/support/routes.rb

```diff
@@ -0,0 +1,38 @@
+require 'support/login_user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.